documentation/src/main/asciidoc/mapstruct-reference-guide.asciidoc

```diff
@@ -252,6 +252,13 @@ Supported values are:
 
 If a policy is given for a specific mapper via `@Mapper#unmappedTargetPolicy()`, the value from the annotation takes precedence.
 |`WARN`
+
+|`mapstruct.
+disableForgedMappingMethodsGeneration`
```

Comment:
I think having it in the annotations is enough... Gunnar has quite some distast for those processor-options anyway 🙂